tv/.env

```diff
@@ -1,3 +1,6 @@
-# CDP
+# CMP
 NEXT_PUBLIC_CMP_PREVIEW_ENDPOINT_URL=
-NEXT_PUBLIC_CMP_PREVIEW_API_KEY=
\ No newline at end of file
+NEXT_PUBLIC_CMP_PREVIEW_API_KEY=
+
+# Website
+NEXT_PUBLIC_WEBSITE_URL=https://www.edge.localhost
```

Comment:
remove https://www.edge.localhost from the value. this will get generated when deploying. This is for local dev only. so never check in a value in end file. also we could get the website url by replacing tv with website in real instances, lets discuss what preferred.

Reply:
Thought I'd use the same pattern as in kiosk with the .env file, but yeah you are right, I'll fix it and update it!